### DISCUSSION TITLE
Question about validation results of FeTS2021

### DISCUSSION THREAD

**Discussion by ddayzzz** [FETS-AI/Challenge#147] [General]
Dear organizer:
I have uploaded the output files to the FeTS2021 validation task of Penn Imaging. 
![image](https://user-images.githubusercontent.com/22568844/158006031-df821d14-d69b-4da6-b6ef-60210f5a5c5a.png)

However, the results are quite strange. It seems that Ground Truth files are missing. 
![image](https://user-images.githubusercontent.com/22568844/158005945-3f1f779b-4ac1-42ce-bfd8-89bf988fb98d.png)
I attached the results below.
[Results-71365227336009964.zip](https://github.com/FETS-AI/Challenge/files/8237134/Results-71365227336009964.zip)

**Comment by sarthakpati** [comment]
Tagging @ujjwalbaid0408

**Comment by ujjwalbaid0408** [comment]
Hi,

It seems you have uploaded the files with the name  BraTS20_Validation_{ID}.nii.gz

For the FeTS challenge, the names of the files must be FeTS21_Validation_{ID}.nii.gz 

Please check and resubmit.

**Reply by ddayzzz** [reply]
Oh, thank you. it seem that I uploaded incorrect files.
